### DISCUSSION TITLE
SMTP not working

### DISCUSSION THREAD

**Discussion by wwinteractive** [berrystephenw/OctoText#143] [Q&A]
I had OctoText working at one point but now can't seem to connect to SMTP. I've tried it with 2 different email accounts (my domain, and gmail). The error I get is "Exception while talking to your mail server, check your SMTP settings."

Octoprint 1.9.3
OctoText 0.3.3.1

I have double checked SMTP settings. For Gmail, I followed the App password instructions.

Added logs from both attempts. 

How else can I trouble shoot this please?

[octoprint_gmail.log](https://github.com/berrystephenw/OctoText/files/14619120/octoprint_gmail.log)
[octoprint_ww.log](https://github.com/berrystephenw/OctoText/files/14619121/octoprint_ww.log)

**Comment by dermaxem** [comment]
seems the same here, was woring fine over years. after some time of not using the printer, doing all updates SMTP-Sending fails with this error:
`2024-05-05 12:38:21,902 - octoprint.plugins.OctoText - ERROR - Exception while talking to your mail server [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1056)
Traceback (most recent call last):
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_OctoText/__init__.py", line 255, in smtp_login_server
    SMTP_server = smtplib.SMTP_SSL(name, port, timeout=5)
`

**Comment by skalamondre** [comment]
So is that it? Is OctoText no longer viable?
